src/main/java/com/cburch/logisim/fpga/hdlgenerator/CircuitHDLGeneratorFactory.java

```diff
@@ -283,7 +283,7 @@ public String getComponentStringIdentifier() {
               OneLine.append(" ");
             }
 
-            Contents.addUnique("   {{assign}} {{1}} {{=}} {{2}}{{3}}{{bracketOpen}}{{4}}{{bracketClose}};",
```

Comment:
You sure this is correct fix? Because if you look back in the file history (before my changes) you will see

```java
if (!Contents.contains(line)) Contents.add(line);
```

which is basically what `addUnique()` does now.

Reply:
basically the addUnique would put in the vhdl/verilog file
{{1}} <= {{2}}{{3}}({{4}});
and the next one:
{{1}}{{2}}
That's why I put add, maybe not the correct solution?

Reply:
In #960 I reworked LineBuffer a bit and also improved `addUnique()`. If that can be merged first, then it should suffice to
do:

```java
Contents.addUnique("   {{assign}} {{1}} {{=}} {{2}}{{3}}{{bracketOpen}}{{4}}{{bracketClose}};",
                OneLine, BusName, TheNets.getNetId(Source.getParentNet()), Source.getParentNetBitIndex());
```

and both positional and paired placeholders should be resolved, then final string checked against buffer contents and only added if not present. Please see if merging #960 first won't solve this issue (also another `addUnique()` below IIRC)

Reply:
We will have in both cases conflicts (meaning first meaning this PR and than yours or the other way around), as we modified both HDL-generators, how do we do this the most intelligent way?

Reply:
Well, I think that you need to toss the coin to see who is going to fix his conflicts :)

Reply:
Basically it does not matter, if you fix the problems in #960 I'll review

Reply:
#960 ready for review.